modules/oe_whitelabel_helper/tests/src/Kernel/Plugin/better_exposed_filters/sort/FloatEndSortWidgetTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_whitelabel_helper\Kernel\Plugin\better_exposed_filters\sort;
+
+use Drupal\Tests\better_exposed_filters\Kernel\BetterExposedFiltersKernelTestBase;
+use Drupal\views\Views;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the FloatEndSort exposed form sort widget rendering.
```

Comment:
Let's use the correct machine name or a human name. FloatEndSort is neither :)
For my taste, we could use a human name and then put a `@covers` doc below for the full machine name.
Also, I am not a friend of the "rendering" at the end of the phrase.

```suggestion
 * Tests theming of the 'Float End Sort' widget for "better exposed filters".
 *
 * @see \Drupal\oe_whitelabel_helper\Plugin\better_exposed_filters\sort\FloatEndSortWidget
```
(not saying this is perfect, feel free to improve)

Reply:
Actually `@covers` would be wrong because we are not really testing the functionality, just the theming in whitelabel. So `@see` is more suitable.

Reply:
Or no, `@covers` actually works.
All we do in this class is add classes to the parent widget.
I don't really know if this is a good idea, but this is not the test's fault. So ok for me.

Reply:
we use coversDefaultClass see https://github.com/openeuropa/europa-search-client/commit/fcf1f46811ea031fd5b6d33d2d55b17289262024

Reply:
From what I understand this is only useful in combination with `@covers` annotations on methods.
See https://phpunit.readthedocs.io/en/9.5/annotations.html#coversdefaultclass
Otherwise we could simply use `@covers` directly.
But no strong opinion.